customer-account-extension/locales/fr.json

```diff
@@ -1,4 +1,5 @@
 {
+  "fullPageTitle": "Page",
   "welcome": "Bienvenue à la cible {{target}} de l'extension d'interface utilisateur du compte client ^_^",
```

Comment:
Not introduced in this PR, but I wonder if these emojis are universally understood. 😅 

`^_^`